File: app/src/main/java/com/example/m4_ejerciciosgit/MainActivity.java
package com.example.m4_ejerciciosgit;

import androidx.appcompat.app.AppCompatActivity;

import android.content.Intent;
import android.os.Bundle;
import android.view.View;
import android.widget.Button;
import android.widget.TextView;

public class MainActivity extends AppCompatActivity {


    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        Button platos = findViewById(R.id.home);
        Button ubicacion = findViewById(R.id.location);
        Button salida = findViewById(R.id.exit);

        salida.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {

           finishAndRemoveTask();
            }
        });
        ubicacion.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {

                Intent intent = new Intent(MainActivity.this, MainActivity3.class);
                startActivity(intent);
            }
        });
        platos.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                //iconos
            /*

             */
                Intent intent = new Intent(MainActivity.this, MainActivity2.class);
                startActivity(intent);
            }
        });

    }

}

File: app/src/main/java/com/example/m4_ejerciciosgit/MainActivity.kt
package com.example.m4_ejerciciosgit

import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity

class MainActivity : AppCompatActivity() {//se omite el public por que es una visibilidad predeterminada ,en class en vez de extends se colocan ":"

    override fun onCreate(savedInstanceState: Bundle?) {//protected void se cambia por override fun
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
    }
//anula el void se cambia por fun
    protected fun sumaBoton() {}
    protected fun restaBoton() {}
    protected fun divisionBoton() {}
    protected fun multiplicacionBoton() {}
}